lib/ts/recipe/passwordless/recipe.ts

```diff
@@ -243,7 +244,12 @@ export default class Recipe extends RecipeModule {
         const appInfo = this.getAppInfo();
 
         let magicLink =
-            appInfo.websiteDomain.getAsStringDangerous() +
+            appInfo
+                .getOrigin({
+                    request: getRequestFromUserContext(input.userContext),
```

Comment:
do not call this function here. Instead, make this function accept a req | undefined and pass that in here. All places where we are calling createMagicLink, need to then pass in the req object. If those places don't have the req object, then they can call getRequestFromUserContext

Reply:
This would make it a breaking change